### DISCUSSION TITLE
How to connect flutter app to local instance

### DISCUSSION THREAD

**Discussion by d-e-h-i-o** [supabase/supabase#10120] [Questions]
Hi, I'm trying to set up a dev environment. 

I followed the steps in the [docs](https://supabase.com/docs/guides/cli/local-development) and have a local instance runnning. However, I'm not sure how to connect my flutter app – which is running locally in the Simulator – to my local instance, since the Studio does not have a field for the `SUPABASE_URL`. I tried out API URL: http://localhost:54321 (from `supabase status`) but it does not work.

Does anyone knows how to do that?

**Comment by dshukertjr** [comment]
Connecting to local host from Flutter apps requires a bit of tweak in general. I hope [this](https://stackoverflow.com/questions/47372568/how-to-point-to-localhost8000-with-the-dart-http-package-in-flutter) stack overflow answer helps.
